torch/_inductor/codecache.py

```diff
@@ -62,6 +62,11 @@ def cache_dir():
     )
 
 
+def remove_cache_dir():
```

Comment:
We could also keep the folder, and just clear the contents, but I kinda prefer this.